tools/accuracy_checker/accuracy_checker/adapters/face_98_landmarks_detection.py

```diff
@@ -0,0 +1,17 @@
+import cv2
+import numpy as np
+
+from ..adapters import Adapter
+from ..config import ConfigValidator, StringField, ConfigError
+from ..preprocessor import ObjectCropWithScale
+from ..representation import Face98LandmarksPrediction
+from ..utils import contains_any
+
+class FaceLandmarksAdapter(Adapter):
+    __provider__ = 'face_landmarks_detection'
+    
+    def process(self, raw, identifiers, frame_meta):
+        result = self._extract_predictions(raw, frame_meta)
+        res = [Face98LandmarksPrediction(identifiers, None, None, result['3851'])]
```

Comment:
it is not robust to use name directly (It may be changed with new version of MO)
```suggestion
        res = [Face98LandmarksPrediction(identifiers, None, None, result[self.output_blob])]
```
or if it is not single output, you can make it a parameter of adapter and specify in config file